### ISSUE TITLE
Context menu commands not working for channels with spaces in their name

### ISSUE THREAD

**Issue by lenorani** [chatty/chatty#53]
Some channels have spaces in their channel names. This causes issues with many of the context menu entries in the "Live Channels" browser.

![2016-05-25_18-17-24](https://cloud.githubusercontent.com/assets/19575223/15547823/cfebf95e-22a5-11e6-81a0-c96648b6d7f8.png)

Example channels include: /faceittv and /eleaguetv.

**Comment by tduva** [comment]
This was fixed. Please tell me if you still have issues with this in version 0.8.2.

**Comment by lenorani** [comment]
The issue first started appearing after updating to 0.8.2 (from 0.8.1) and is still present after re-downloading 0.8.2.

However since you mentioned that it has already been fixed I also tried the newest beta release (0.8.3b2) and there the issue seems to be fixed. In light of this, I think this issue can be considered to be resolved.

(I should probably have tried the beta version before reporting this issue and will do so in the future.)
